changelog.md

```diff
@@ -10,6 +10,18 @@ _Nothing yet.._
 - react-native-bpk-component-icon: 1.1.15 => 1.2.0
  - `icons` object is now exposed, allowing users to refer to icon names using a constant.
 
+**Breaking:**
+- bpk-component-banner-alert:
+  - Removed `dismissable` property in favour of `configuration`.
+  - A banner alert with `children` will no longer be auotmatically expandable. Use the newly introduce `configuration` prop instead.
+
+**Added:**
+- bpk-component-banner-alert:
+  - Introduced new component for the different banner alert configurations.
```

Comment:
The grammar is shit here. Need to rewrite

Reply:
@shaundon is the man for the job!